chapters/11-version-control-workflow.Rmd

```diff
@@ -2,12 +2,28 @@
 
 - Make use of R builtin tools like the usethis pull request helpers
   (and learning how to make use of this yourself)
+- E.g. with pr_* functions from usethis
 - Using this workflow as a team or as a contributor
 
-
-
 ## Final exercise
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+Complete these tasks for your package for the first project assignment.
```

Comment:
"Complete these tasks as part of the package for your first project assignment" would be my suggestion on how to phrase this.